docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,63 @@
+---
+title: .NET Core distribution packaging
```

Comment:
I'm not sure I understand. Do you mean append that to the title? Is it something we're doing to all topics now? If so, shouldn't the tooling do that for us?